modules/n1ql/pages/n1ql-language-reference/prepare.adoc

```diff
@@ -234,96 +231,13 @@ include::ROOT:partial$query-context.adoc[tag=section]
 .Query
 [source,sqlpp]
 ----
-PREPARE SELECT * FROM route
-WHERE airline = "FL";
+include::example$n1ql-language-reference/prepare-q1.n1ql[]
```

Comment:
This directory is for DDL examples, can you put this example in the `utility` directory?